src/assets/sitemapper.js

```diff
@@ -109,7 +111,10 @@ export default class Sitemapper {
           return xmlParse(response.body);
         })
         .then(data => resolve({ error: null, data }))
-        .catch(response => resolve({ error: response.error, data: {} }));
+        .catch(response => {
+          console.log(response);
```

Comment:
Just revert this part, rebase to have tests working again, and should be good to merge